contracts/test/crosschain/RuleEntitlement.t.sol

```diff
@@ -115,6 +121,17 @@ contract RuleEntitlementTest is TestUtils, IEntitlementBase {
     assertEq(ruleOperations.length, 3);
   }
 
+  function test_setRuleEntitlementTwice() external givenRuleEntitlementIsSet {
```

Comment:
May be a good idea to also test that the previous entitlementSet was deleted along with the final cardinality of the set.